packages/mdc-ripple/README.md

```diff
@@ -74,41 +75,17 @@ use [mdc-elevation](https://github.com/material-components/material-components-w
   text-align: center;
   /* Indicate to user element is interactive. */
   cursor: pointer;
-
-
-  /* Use the surface's ::before pseudo-element as a basic interaction indicator */
-
-  &::before {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
-    pointer-events: none;
-    background-color: black;
-    transition: opacity 110ms ease;
-    will-change: opacity;
-  }
-
-  &:active::before {
-    opacity: .18;
-  }
-
-  &:focus::before {
-    opacity: .06;
-  }
-}
 ```
 
 #### Adding the ripple Sass
 
-When a ripple is successfully initialized on an element, it dynamically adds a `mdc-ripple-upgraded` class to that element. Therefore, to add a ripple to our surface, first we include the proper Sass mixins within our surface's styles when it contains this class. We also add a few additional properties that ensure the ripple's UX is correct.
+To add a ripple to our surface, first we include the proper Sass mixins within our surface's styles when it contains this class. We also add a few additional properties that ensure the ripple's UX is correct.
```

Comment:
I think we can remove the portion of text that says "when it contains this class", since I'm pretty sure _"this class"_ refers to `mdc-ripple-upgraded`, which we now take care of automatically.